package.json

```diff
@@ -55,6 +55,7 @@
   },
   "contributors": [
     "David Dias <daviddias.p@gmail.com>",
-    "Friedel Ziegelmayer <dignifiedquire@gmail.com>"
+    "Friedel Ziegelmayer <dignifiedquire@gmail.com>",
+    "Richard Schneider <makaretu@gmail.com>"
```

Comment:
this is not needed, its added automatically on release.

Reply:
ok will reset